src/inmanta/deploy/scheduler.py

```diff
@@ -68,17 +68,17 @@ class StaleResource(Exception):
 
 
 @dataclass(frozen=True)
-class ResourceIntent:
+class ResourceVersion:
```

Comment:
rename 2.

motivation: conflict with proposed rename of `deploy.state.ResourceDetails` -> `deploy.state.ResourceIntent`

Reply:
I find `ResourceVersion` a strange name, because it represents a log more than just a version. What about `ResourceVersionIntent` in analogy with `ResourceId` and `ResourceVersionId`?

Reply:
I'm fine with that. Part of the reason for leaving the `Intent` part off was verbosity, part was to more clearly differentiate between this class and its `intent` attribute. But neither of those arguments are very strong.

Reply:
@wouterdb I'm siding with @arnaudsjs here. Do you agree with `ResourceVersionIntent` for this one?